src/Extensions/DocumentSelector.re

```diff
@@ -11,7 +11,15 @@ type item = {language: string};
 type t = list(item);
 
 let matches = (v: t, fileType: string) => {
-  List.exists(item => item.language == fileType, v);
+  List.exists(item => String.equal(item.language, fileType), v);
+};
+
+let toString = (v: t) => {
+  "DocumentSelector("++ 
+  List.fold_left((prev, curr) => {
+    prev ++ "," ++ curr.language
+    }, "", v)
```

Comment:
This would be simpler as `List.map(x => x.language) |> String.concat(",")` I think.

Also, the convention is to use `acc` (short for "accumulator") for `prev` here. `prev` seems to me to imply that it has the same type as `curr`.

Reply:
Good call! Makes sense about `acc` vs `prev` - I'll switch to that convention.